core/io/packed_data_container.h

```diff
@@ -36,7 +36,7 @@
 class PackedDataContainer : public Resource {
 	GDCLASS(PackedDataContainer, Resource);
 
-	enum {
+	enum : uint32_t {
```

Comment:
I assume compilation fails without this? Is it absolutely needed?

Reply:
It fixes warnings that would otherwise be produced when building with VS clang.